### PR TITLE
Portfolio Project Mobile View Phase 1 - Introduction Section

### PR DESCRIPTION
**Pull request by mudasiri**
**Portfolio Website Mobile First Design**
In this Phase i designed the introduction section of my portfolio website which include the following sections:

1. Header section with Logo and Hamburger Icon
2. An about me Section with a welcome message and links to my social Media accounts

The entire code is designed for view only

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,68 @@
...
+
+### Usage
+NA
+### Run tests
+so when you make a pull request there will be some tests that your code should pass ensure that your code passes all of the test.
+### Deployment
+NA
+
+
+## Authors
```
Comment (Qoosim) [inline README.md:38]: - [ ] Kindly change the `Authors` to `Author` as this is a solo project which is supposed to carry single author.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to My Portfolio Website</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/69bb18a36d.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header class="flexbox">
+        <span class="logo">Mudasiri</span>
```
Comment (Qoosim) [inline index.html:13]: - [ ] Kindly give your logo a link (`a` tag). It is for these three reasons: convention, semantics, and user experience(UX). You can check this [link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info

File: index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        <nav>
+            <i class="fa-solid fa-bars"></i>
+        </nav>
+    </header>
+    <section class="intro">
+        <h1 class="title">I’m Mudasir
+            Welcome to my Site!</h1>
+        <p class="description">I’m a fullstack web developer! I can help you build a product website, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <h3 class="action-text">LET’S CONNECT</h3>
```
Comment (Qoosim) [inline index.html:22]: - [ ] Kindly make your let's connect a `button` or `a` tag for semantics reason.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        <ul class="social-media">
+            <li><i class="fa-brands fa-twitter"></i></li>
+            <li><i class="fa-brands fa-linkedin"></i></li>
+            <li><i class="fa-brands fa-github"></i></li>
+            <li><i class="fa-brands fa-medium"></i></li>
```
Comment (Qoosim) [inline index.html:27]: - [ ] Kindly make your list items the links  (i.e `a` tag) to your social media pages. This is for the semantics reason and accessibility purpose

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @mudasiri, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
 -   No linter errors. :heavy_check_mark: 
 -   Professional README file. :heavy_check_mark: 
 -   Meaningful commit messages. :heavy_check_mark: 
 -   Correct GiHub Flow. :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] You've put in a lot of effort for pushing your portfolio this far and I must commend your effort for that. However, there're few things you need to work on before moving to the next milestone. Your chosen template has a background image which you are expected to implement in this milestone. This should be achieved with the use of css background-image property, not with the `img` tag. At this moment, your project does not have that background image. Kindly look into your chosen template and implement that background image decoration.
- [ ] It is required to produce a close replica of any chosen template by the student. This includes, font-size, font-type, font-weight, color(text and background), line-space, margin, and padding among others. Let try as much as we can to produce a close replica of our chosen template. Kindly look at the your present design and compare it with your chosen figma template.

### Your Design

your-design


### Figma Design

figma-design

File: style.css [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  list-style-type: none;
+  padding-left: 0;
+}
+
+.fa-brands {
+  color: #505f79;
+}
+
+.title {
+  font-style: normal;
```
Comment (sajid-munawar) [inline style.css:77]: - [ ] As the previous reviewer mentioned please make sure you stick to the `Figma`  here you can use `font-style: bold`.
your design 👇
portfigma

Figma design 👇
figma1

File: style.css [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
```
Comment (sajid-munawar) [inline style.css:81]: - [ ]  Please make sure you use `color: #172B4D` instead of  `color: #172b4d`

File: style.css [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+.social-media li {
+  display: inline;
+}
+
+.social-media {
+  list-style-type: none;
+  padding-left: 0;
+}
```
Comment (sajid-munawar) [inline style.css:70]: - [ ] Please make sure you give a gap between social media icons.

your design👇
gap

Figma design 👇
figmsocial

**Review by sajid-munawar — CHANGES_REQUESTED** [review]
Hi @mudasiri , 

_Great job so far 👏 💯 👍. You've done well implementing some of the requested changes. However, there are still some which aren't addressed yet. Please, check the comments below._
 

## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed some of the issues raised from the previous review. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
  
  - N/A 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by ichala — APPROVED** [review]
Hi @mudasiri  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implement✔️

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


*************************************
# Optional Suggestions 

- [ ] N/A


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**